src/simple_cache_client.rs

```diff
@@ -168,11 +209,12 @@ impl SimpleCacheClientBuilder {
         );
         let data_client = ScsClient::new(data_interceptor);
 
+        #[allow(deprecated)]
```

Comment:
What is deprecated in here?

Reply:
This was left over from a mid-refactoring change.